test/suites/alpha/integration/extended_resources_test.go

```diff
@@ -272,6 +273,123 @@ var _ = Describe("Extended Resources", func() {
 		env.ExpectCreatedNodeCount("==", 1)
 		env.EventuallyExpectInitializedNodeCount("==", 1)
 	})
+	It("should provision nodes for a deployment that requests aws.amazon.com/neuron", func() {
```

Comment:
Will some of these tests fail for < 1.28 if the AMI doesn't have the device driver shipped with the image by default?

Reply:
I'd have to check, I don't believe the driver was added in the 1.28 AMI though (but I could be mistaken). This will need to be scoped to whichever versions have it.